spec/classes/puppet_server_config_spec.rb

```diff
@@ -79,7 +79,7 @@
             :creates => "#{ssldir}/certs/puppetmaster.example.com.pem",
             :command => "#{puppetcacmd} --generate puppetmaster.example.com",
             :require => ["Concat[#{conf_file}]", "Exec[puppet_server_config-create_ssl_dir]"],
-          }).that_notifies('Service[httpd]')
```

Comment:
> I wonder if it ought to do the same for Puppet server too.

Sorry, I think this was already the case, now I look at the code in more detail!